com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/common/ModelController.java

```diff
@@ -27,55 +33,99 @@
 import com.eclipsesource.modelserver.emf.configuration.ServerConfiguration;
 import com.google.inject.Inject;
 
-import io.javalin.Context;
 import io.javalin.apibuilder.CrudHandler;
-import io.javalin.json.JavalinJackson;
+import io.javalin.http.BadRequestResponse;
+import io.javalin.http.Context;
+import io.javalin.http.Handler;
+import io.javalin.plugin.json.JavalinJackson;
 
 public class ModelController implements CrudHandler {
 
+	private static final Logger LOG = Logger.getLogger(ModelController.class);
+
 	@Inject
 	private ResourceManager resourceManager;
 	@Inject
 	private ServerConfiguration serverConfiguration;
 
-	private ResourceSet resourceSet= new ResourceSetImpl();
+	private ResourceSet resourceSet = new ResourceSetImpl();
+	private Map<URI, EObject> modelRepository = new HashMap<URI, EObject>();
 	
 	public ModelController() {
 		JavalinJackson.configure(EMFJsonConverter.setupDefaultMapper());
 	}
 
 	@Override
 	public void create(Context ctx) {
-		// TODO Auto-generated method stub
+		try {
+			EObject model = ctx.bodyAsClass(EObject.class);
+			
+			EStructuralFeature name = model.eClass().getEStructuralFeature("name");
```

Comment:
As discussed, the name should become a property in the request, would you mind creating an issue for that? We'll fix this in a follow-up PR then.